lib/vm.js

```diff
@@ -97,6 +100,19 @@ function getContextOptions(options) {
   return {};
 }
 
+function isContext(sandbox) {
+  if (arguments.length < 1) {
```

Comment:
Do we need this one though? If sandbox is undefined, it will show up as a ERR_INVALID_ARG_TYPE as well, it seems rare that someone would just do “isContext()”